### ISSUE TITLE
Add commands to sandbox with a 5-second cooldown between commands & certain commands capable of crashing such as spawning water restricted to admins.

### ISSUE THREAD

**Issue by BACONOCAB** [transhumandesign/kag-base#225]
What the title says.

**Comment by Vikhenzo** [comment]
Yes I agree some commands should be enabled in sandbox for everyone

**Comment by FuzzCat927** [comment]
Currently `!spawnwater` and `!bot` are restricted to admins, but doesn't "check seclevs", just uses the `CPlayer.isMod` method

**Comment by 1bardesign** [comment]
Modification to check seclevs in chat commands is welcome. A sandbox-specific command set could possibly be added as well (just check the gamemode and enable them without `sv_test`).
